waku/v2/dnsdisc/metrics.go

```diff
@@ -0,0 +1,51 @@
+package dnsdisc
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/waku-org/go-waku/waku/v2/utils"
+)
+
+var dnsDiscoveredNodes = prometheus.NewCounter(
+	prometheus.CounterOpts{
+		Name: "waku_dnsdisc_discovered",
+		Help: "The number of nodes discovered via DNS discovery",
```

Comment:
I agree. It would certainly be much more informative than having a single sum of all discovered peers.
Added https://github.com/waku-org/go-waku/issues/652 to track this work as it would be a new feature out of the scope of this refactor